boards/ek-lm4f120xl/include/periph_conf.h

```diff
@@ -99,6 +101,68 @@ extern "C" {
 #define ADC_NUMOF           (12)
 /** @} */
 
+/**
+ * @name SPI configuration
+ * @{
+ */
+static const struct spi_conf_s spi_confs[] = {
```

Comment:
I can easily typedef the struct, call it spi_conf_t and be closer to what is already done in other ports. That would be easier to read.